CONTRIBUTING.md

```diff
@@ -27,25 +27,28 @@ If you are hired for an Upwork job and have any job-specific questions, please a
 If you've found a vulnerability, please email security@expensify.com with the subject `Vulnerability Report` instead of creating an issue.
 
 ## Payment for Contributions
-We hire and pay external contributors via Upwork.com. If you'd like to be paid for contributing, please create an Upwork account and apply for a job in the [Upwork issue list](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2). If you think your compensation should be increased for a specific job, you can request a reevaluation by commenting in the Github issue where the Upwork job was posted. Payment for your contributions will be made no less than 7 days after the pull request is deployed to production to allow for regression testing. We hire one contributor for each Upwork job. New contributors are limited to working on one job at a time, however experienced contributors may work on numerous jobs simultaneously. If you have not received payment after 8 days of the PR being deployed to production, please email contributors@expensify.com referencing the GH issue and your GH handle.
```

Comment:
I noticed you deleted the sentence `We hire one contributor for each Upwork job.`, I assume this is because we now may "technically" hire 2 contributors for one job - if one contributor reported the issue and a second contributor fixed the issue, is that correct? If so, I still think it's fine to keep this sentence, or maybe clarify like this: `We hire one contributor to work on each Upwork job.`

Reply:
Yeah, I removed cuz we'll technically be hiring multiple because of the need to pay the proposer.  If you think it's helpful to have `We hire one contributor to work on each Upwork job.` in there, please add it back.

Reply:
Hmm I think it's find without that sentence actually 👍